users/validations.py

```diff
@@ -0,0 +1,36 @@
+import re
+
+def validate_account(value):
+    account_reg = r"(^[a-zA-Z0-9]+.{6,15}$)"
+    regex = re.compile(account_reg)
+    if regex.match(value):
+        return True
+    else:
+        return False
```

Comment:
```suggestion
    if regex.match(value):
        return True
    return False
```
`else`가 없어도 바로 위의 `if`문이 만족되지 않는다면 바로 도달할 수 있는 로직입니다. 꼭 필요한 경우가 아니면 `else`를 사용하지 않는 것이 보다 파이썬다운 코딩 스타일입니다. `else`는  지워주셔도 됩니다 :)